<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Console</class>
 <widget class="QWidget" name="Console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTextEdit" name="outText">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Console_Send">
     <item>
      <widget class="MCmdLine" name="inText"/>
     </item>
     <item>
      <widget class="QPushButton" name="bManual">
       <property name="text">
        <string>console</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="StatusPanel">
     <item>
      <widget class="QToolButton" name="bConnectStatus">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/CLOSE.svg</normaloff>
         <normalon>:/res/CHECK MARK.svg</normalon>:/res/CLOSE.svg</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status_text">
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>boot</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MCmdLine</class>
   <extends>QLineEdit</extends>
   <header location="global">mcmdline.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bConnectStatus</sender>
   <signal>toggled(bool)</signal>
   <receiver>Console</receiver>
   <slot>setConnect_status(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>14</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setConnect_status(bool)</slot>
 </slots>
</ui>
